libs/sdl/gl.c

```diff
@@ -421,7 +439,14 @@ HL_PRIM vdynamic *HL_NAME(gl_create_renderbuffer)() {
 
 HL_PRIM void HL_NAME(gl_bind_renderbuffer)( int target, vdynamic *r ) {
 	GLOG("%d,%d",target,ZIDX(r));
+#if	defined(HL_IOS) || defined(HL_TVOS)
+	SDL_SysWMinfo info;
+	SDL_VERSION(&info.version);
+	SDL_GetWindowWMInfo(SDL_GL_GetCurrentWindow(), &info);
+	glBindRenderbuffer(GL_RENDERBUFFER, ZIDX(r) == 0 ? info.info.uikit.colorbuffer : ZIDX(r));

```

Comment:
Same remark as above, let's try to reduce SDL calls.

Reply:
Fixed in https://github.com/HaxeFoundation/hashlink/pull/101/commits/1af597801d7ba76741dd36435f13b33acf9bbe16